messaging/src/main/java/org/axonframework/deadline/quartz/QuartzDeadlineManager.java

```diff
@@ -356,6 +360,19 @@ public Builder refireImmediatelyPolicy(Predicate<Throwable> refireImmediatelyPol
             return this;
         }
 
+        /**
+         * Sets the {@link MessageNameResolver} used to resolve the {@link QualifiedName} when publishing {@link EventMessage EventMessages}.
```

Comment:
Darn, not sure if I suggested this wrongly now, but regardless, the JavaDoc should be about `DeadlinMessages`, not `EventMessages` :-[